src/steamship/agents/tools/video_generation/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .did_video_generator_tool import DIDVideoGeneratorTool
+
+__all__ = [
+    "DIDVideoGeneratorTool",
```

Comment:
random nonsense aside: i find `DID` disconcerting. i would like to speak to their branding folks!